src/main/java/org/sqlite/SQLiteJDBCLoader.java

```diff
@@ -62,9 +62,46 @@
      * @return True if SQLite native library is successfully loaded; false otherwise.
      */
     public static boolean initialize() throws Exception {
-        loadSQLiteNativeLibrary();
-        return extracted;
+        // only cleanup before first extract
+		if(!extracted) {
+			cleanup();
+		}
+		loadSQLiteNativeLibrary();
+		return extracted;
     }
+	
+	
+	/**
+	* Deleted old nativ libraries e.g. on Windows this are not removed on VM-Exit (bug #80)
+	*/
+	static void cleanup(){
+
+		final String ver = org.sqlite.SQLiteJDBCLoader.getVersion();
+		String tempFolder = new	java.io.File(System.getProperty("java.io.tmpdir")).getAbsolutePath();					
+		java.io.File dir = new java.io.File(tempFolder);
+		java.io.File [] files = dir.listFiles(new java.io.FilenameFilter() {
+			@Override
+			public boolean accept(java.io.File dir, String name) {
+				return name.startsWith("sqlite-" + ver);
+			}
+		});
+		
+		for (java.io.File tempFile : files) {
+			if(tempFile.getName().indexOf(".lck") > 0) { continue; }
+			java.io.File lckFile = new java.io.File(tempFile.getName() + ".lck");
+			if(!lckFile.exists()) {
+				try{
+					tempFile.delete();
+				}
+				catch(SecurityException ex){
+					
```

Comment:
Swallowing exception is a bad habit. At least we should show the error message here.